### PR TITLE
Add getAttachments function

### PR DESCRIPTION
**Pull request by OzzyCzech**
Add getAttachments function for getting all email attachments. It's useful if you use different way for email sending e.g. via Mailgun you are not able fill attachment parameter.

https://documentation.mailgun.com/api-sending.html#sending

### CODE REVIEW

File: src/Mail/Message.php [hunk 1]
```diff
@@ -300,6 +300,13 @@ public function addAttachment($file, $content = NULL, $contentType = NULL)
 		return $this->attachments[] = $this->createAttachment($file, $content, $contentType, 'attachment');
 	}
 
+	/**
+	 * Return list of all attachments
+	 * @return Array
+	 */
+	public function getAttachments() {
```
Comment (fprochazka) [inline src/Mail/Message.php]: can you please fix the coding style?

Comment (OzzyCzech) [inline src/Mail/Message.php]: done

File: src/Mail/Message.php [hunk 2]
```diff
@@ -300,6 +300,14 @@ public function addAttachment($file, $content = NULL, $contentType = NULL)
 		return $this->attachments[] = $this->createAttachment($file, $content, $contentType, 'attachment');
 	}
 
+	/**
+	 * Return list of all attachments
+	 * @return Array
```
Comment (dg) [inline src/Mail/Message.php]: Returns & dot.

File: src/Mail/Message.php [hunk 3]
```diff
@@ -300,6 +300,14 @@ public function addAttachment($file, $content = NULL, $contentType = NULL)
 		return $this->attachments[] = $this->createAttachment($file, $content, $contentType, 'attachment');
 	}
 
+	/**
+	 * Return list of all attachments
```
Comment (dg) [inline src/Mail/Message.php]: Returns & dot.

Comment (OzzyCzech) [inline src/Mail/Message.php]: done

File: src/Mail/Message.php [hunk 4]
```diff
@@ -300,6 +300,14 @@ public function addAttachment($file, $content = NULL, $contentType = NULL)
 		return $this->attachments[] = $this->createAttachment($file, $content, $contentType, 'attachment');
 	}
 
+	/**
+	 * Returns list of all attachments.
+	 * @return Array
```
Comment (fprochazka) [inline src/Mail/Message.php]: this should be `@return MimePart[]`

File: src/Mail/Message.php [hunk 5]
```diff
@@ -300,6 +300,14 @@ public function addAttachment($file, $content = NULL, $contentType = NULL)
 		return $this->attachments[] = $this->createAttachment($file, $content, $contentType, 'attachment');
 	}
 
+	/**
```
Comment (matej21) [inline src/Mail/Message.php]: There should be two blank lines between methods

Comment (OzzyCzech) [inline src/Mail/Message.php]: done

File: src/Mail/Message.php [hunk 6]
```diff
@@ -302,6 +302,16 @@ public function addAttachment($file, $content = NULL, $contentType = NULL)
 
 
 	/**
+	 * Returns list of all attachments.
+	 * @return Array
```
Comment (fprochazka) [inline src/Mail/Message.php]: this should be `@return MimePart[]`

Comment (OzzyCzech) [inline src/Mail/Message.php]: done

**Comment by dg** [comment]
The last thing: try to write better commit message please

**Comment by dg** [comment]
This is not consistent commit message…

**Pull request closed without merging**